conans/client/remote_registry.py

```diff
@@ -194,13 +194,14 @@ def exists_function(remotes):
                                      % remote_name)
         self._add_update(remote_name, url, verify_ssl, exists_function, insert)
 
-    def remove(self, remote_name):
+    def remove(self, remote_name, force=False):
         self._remotes = None  # invalidate cached remotes
         with fasteners.InterProcessLock(self._filename + ".lock", logger=logger):
             remotes, refs = self._load()
-            if remote_name not in remotes:
+
+            if not remotes.pop(remote_name, force):
```

Comment:
Probably it would be very nice to show a warning that the remote doesn't exist. Please use ``self.output.warn()``.